CONTRIBUTING.md

```diff
@@ -20,8 +20,8 @@ fixes, documentation, examples... But first, read this page (including the small
     + [Eclipse Setup](#eclipse-setup)
     + [IDEA Setup](#idea-setup)
       - [How to work](#how-to-work)
-      - [`OutOfMemoryError` while importing](#-outofmemoryerror--while-importing)
-      - [`package sun.misc does not exist` while building](#-package-sunmisc-does-not-exist--while-building)
+      - [`OutOfMemoryError` while importing](#outofmemoryerror-while-importing)
+      - [`package sun.misc does not exist` while building](#package-sunmisc-does-not-exist-while-building)
```

Comment:
I would refrain from doing any changes to the TOC as it's automatically generated.